RPIPs/RPIP-X.md

```diff
@@ -0,0 +1,308 @@
+
+## Motivation
+    
+This informational document seeks to clarify the existing governance structure and delineate the powers and authorities of established entities clearly, to ensure transparency and efficiency in Rocket Pool’s decision-making process. We believe doing so will also positively affect:
+- Ease of onboarding new contributors to Rocket Pool Governance.
+- General knowledge about the protocol and how it operates.
+- Ability to identify stakeholders and their responsibilities.
+
+In short, having better baseline information improves the quality of contributor life for the protocol, and establishing this work as an informational RPIP would make it easier to access and trust the validity of the described areas.
+
+## Specification
+
+### Definitions
+
+There are several terms used in this document that are worth defining explicitly to avoid misunderstanding and ambiguity.
+
+#### Principal
+
+An individual or entity that can be said to ‘own’ the protocol.
+
+#### Agent
+
+An individual or entity that provides some service to the principals, or holds power delegated by the principals.
+
+#### Power
+
+Power in this document is used to refer to the raw capability to do something. This refers to explicit permissions granted in the protocol’s smart contracts, administrative access to software platforms, and so on.
+
+A principal or agent has the power to perform action X when every limitation to their role in performing action X is self-applied, rather than being applied by another party.
+
+To give an example, the oDAO has the power to make changes to the Rocket Pool Protocol due to the permissions trustedNodes (in aggregate) hold in the protocol contracts, and because the limitations to the oDAO ratified by the pDAO are self-applied by the oDAO.
+
+Best expressed in the phrase: ‘Are we able to do X?’
+
+#### Authority
+
+Authority in this document is used to refer to the right to do something. This refers to the implicit and explicit decisions made by the community at large as to ‘who do we want to do X’.
+
+Authorities are a bit complex in that they often stem from a string of decision-making powers, some explicit and some explicit. This document seeks to document what clear authorities exist based on approved governance actions and their ramifications, but may not be able to capture the full intricacy of who has the "right" to take an action.
+
+A principal or agent has the authority to do X if the Rocket Pool community has explicitly or implicitly agreed that that principal or agent is allowed or supposed to be doing X.
+
+To give an example, the pDAO has the authority to make changes to the Rocket Pool Protocol due to the shared understanding of the community that registeredNodes are the ‘principals’ of the Rocket Pool protocol.
```

Comment:
I like the power example a lot, I'm less sold on the authority side. I tried updating the language to convey it a little more clearly, not sure if you think this addresses this point however.

For authority the main thing I want to preserve is that it's based on shared understanding rather than physical access. I think using the same price setting bit could be confusing there and instead convey that authority is the responsibility to do something rather than the shared permission to take action.